source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3628,9 +3745,75 @@ export class AdaptiveCard extends ContainerWithActions {
         return renderedCard;
     }
 
+    updateLayout(processChildren: boolean = true) {
+        super.updateLayout(processChildren);
+        this.handleBottomOverflows();
+    }
+
     canContentBleed(): boolean {
         return true;
     }
+
+    private getChildElements(): Array<CardElement> {
```

Comment:
There shouldn't be a method that needs to understand the internals of various element types. This doesn't work with extensibility; a developer won't be able to add a new element type that itself has children, because this method won't know how to gather its children.

Reply:
That's a good point. As it is now, new element types would be entirely hidden if they overflow the card, so I was originally thinking developers would override `truncateOverflows()` and `resetOverflowTruncation()` methods if their custom element has children -- but then I realized we don't need to treat that as a special case. I'll work on moving the logic into the `Container` and `ColumnSet` classes.